useful-scripts/docs/setup-environment_mac.md

```diff
@@ -0,0 +1,14 @@
+# Computer Environment Set Up for Macs
+
+Download the `setup environment mac script` from your terminal with the following command:
+
+```bash
+curl https://raw.githubusercontent.com/Wolox/tech-guides/master/useful-scripts/scripts/setup-environment_mac.sh > setup-environment_mac.sh
+```
+
+And execute it:
+
+```bash
+chmod +x setup-environment_mac.sh
```

Comment:
This name differs from the actual name of the file. It has a "-" instead of "_"

´setup-environment_mac.sh´> ´setup_environment_mac.sh´

@danielaRiesgo @hdf1986 @pablog

Reply:
I think will be a good idea to open an issue with this (or submit a new PR with the fix, because this one is already merged haha)

Reply:
K, I'll make another PR